lib/web_ui/lib/src/engine/window.dart

```diff
@@ -648,16 +649,10 @@ class EngineWindow extends ui.Window {
     bool? useSingle,
```

Comment:
why make this nullable? we can achieve the same behavior by default to false or always ask for this parameter